android/src/main/java/io/sentry/react/RNSentryModuleImpl.java

```diff
@@ -1,5 +1,6 @@
 package io.sentry.react;
 
+import android.view.Choreographer;
```

Comment:
I think it is safe to remove this import since the class is only used inside the `RNSentryTimeToDisplay` implementation.
```suggestion
```

Reply:
I've also noticed a few more imports unrelated with this PR that can be safely removed. I'm attaching a patch since it escapes the scope GH allows me to comment on. I can handle this on a separate PR if it is more proper.
[imports.patch](https://github.com/user-attachments/files/17113794/imports.patch)

Reply:
Thank you for checking this!
Would you mind making another PR for the other unused imports? (this way you also get another PR here hehe)

Reply:
> Would you mind making another PR for the other unused imports? (this way you also get another PR here hehe)

@lucas-zimerman Closing the loop here I've opened a PR at https://github.com/getsentry/sentry-react-native/pull/4119
Feel free to merge on top of your PR or after your PR is merged.